lib/core/src/model.rs

```diff
@@ -501,6 +502,10 @@ pub enum SdkEvent {
     },
     /// Synced with mempool and onchain data
     Synced,
+    /// Failed to sync with mempool
```

Comment:
Nit: just to keep it matched with the `Synced` event
```suggestion
    /// Failed to sync with mempool and onchain data
```